### ISSUE TITLE
Unexpected character encountered while parsing value A

### ISSUE THREAD

**Issue by OliverOffing** [pauldotknopf/twitter-dump#18]
```
# ./twitter-dump search -q "geocode:52.5186202,13.3761872,0.2km" -o test.json
[20:02:47 INF] Requesting https://api.twitter.com/2/search/adaptive.json?include_profile_interstitial_type=1&include_blocking=1&include_blocked_by=1&include_followed_by=1&include_want_retweets=1&include_mute_edge=1&include_can_dm=1&include_can_media_tag=1&skip_status=1&cards_platform=Web-12&include_cards=1&include_composer_source=true&include_ext_alt_text=true&include_reply_count=1&tweet_mode=extended&include_entities=true&include_user_entities=true&include_ext_media_color=true&include_ext_media_availability=true&send_error_codes=true&simple_quoted_tweets=true&tweet_search_mode=live&count=100&query_source=typed_query&pc=1&spelling_corrections=1&ext=mediaStats,highlightedLabel,cameraMoment&q=geocode%3A52.5186202,13.3761872,0.2km..
Unhandled exception. Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: A. Path '', line 1, position 1.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonWriter.WriteToken(JsonReader reader, Boolean writeChildren, Boolean writeDateConstructorAsDate, Boolean writeComments)
   at Newtonsoft.Json.Linq.JTokenWriter.WriteToken(JsonReader reader, Boolean writeChildren, Boolean writeDateConstructorAsDate, Boolean writeComments)
   at Newtonsoft.Json.JsonWriter.WriteToken(JsonReader reader, Boolean writeChildren)
   at Newtonsoft.Json.JsonWriter.WriteToken(JsonReader reader)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateJToken(JsonReader reader, JsonContract contract)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objec
```

**Comment by adamhynek** [comment]
I only just found this tool, and had the same issue. The problem is that twitter is compressing the response, which I guess they didn't used to do. It's a simple fix, #19 does it for me.

**Comment by pauldotknopf** [comment]
[Pushed](https://www.nuget.org/packages/TwitterDump/0.5.0), thanks!
